Added the product reviews api to state for the product id

noginger13 opened the pull request:
This should add the product reviews api to the state without breaking anything.

noginger13 committed:
intial branch commit

noginger13 committed:
added the product reviews api to state for the product id

brandon-hester reviewed (changes requested):
in productScreen.js add the loading logic

noginger13 committed:
resolved merge conflicts in files that were merged

Merge branch 'dev' of https://github.com/void2205/Void-Front-End-Capstone into rating
